unittests/test-contracts/db_find_secondary_test/CMakeLists.txt

```diff
@@ -0,0 +1,2 @@
+configure_file( ${CMAKE_CURRENT_SOURCE_DIR}/db_find_secondary_test.wasm ${CMAKE_CURRENT_BINARY_DIR}/db_find_secondary_test.wasm COPYONLY )
+configure_file( ${CMAKE_CURRENT_SOURCE_DIR}/db_find_secondary_test.abi  ${CMAKE_CURRENT_BINARY_DIR}/db_find_secondary_test.abi  COPYONLY )
```

Comment:
Can we set this up like the other test directories where it compiles if `EOSIO_COMPILE_TEST_CONTRACTS` is set?

Reply:
Yeah that crossed my mind... I was sloppy about that over here on this one too
https://github.com/AntelopeIO/spring/blob/f871bf2c6356cc6273473aa2ab0ac8ddcf7f4f21/unittests/contracts/jumborow/CMakeLists.txt#L1-L2

Reply:
Will do what's in #1026 here if that one is approved